lbrynet/metadata/metadata_schemas.py

```diff
@@ -0,0 +1,269 @@
+VER_001 = {
+    '$schema': 'http://json-schema.org/draft-04/schema#',
+    'title': 'LBRY metadata schema 0.0.1',
+    'definitions': {
+        'fee_info': {
```

Comment:
The fee_info sub-schema is duplicated from Fee.py. If we found a way to use JSON files instead of hard coding it in Python, we could use JSON Schema's `$ref` keyword to pull in the schema from the other file instead.